tests/scripts/local_monitoring_tests/json_files/datasources.json

```diff
@@ -0,0 +1,86 @@
+[
+    {
```

Comment:
should we also have Cloud Provider name :   like AWS, EKS , Openshift etc ?

Reply:
Right now, we only support prometheus. Anything except that will result into UnSupportedDataSourceProvider exception.